src/compas/files/base_reader.py

```diff
@@ -18,85 +18,164 @@ class BaseReader(object):
     ----------
     location : str or pathlib object
         Path or URL to the file
-    """
+    is_url
+
+    Methods
+    -------
```

Comment:
this is not necessary because sphinx does this automatically

Reply:
Ok!